Source/Array.swift

```diff
@@ -55,6 +55,34 @@ public func -<< <T, U>(f: T -> [U], a: [T]) -> [U] {
 }
 
 /**
+    compose two functions that produce arrays of values, from left to right
+
+    produces a function that applies that flatMaps the second function over each element in the result of the first function
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 124 characters (line_length)